src/vmm/src/cpuid/transformer/common.rs

```diff
@@ -11,6 +11,10 @@ use crate::cpuid::common::get_cpuid;
 // CPUID bits in ebx, ecx, and edx.
 const EBX_CLFLUSH_CACHELINE: u32 = 8; // Flush a cache line size.
 
+// The APIC ID shift in leaf 0xBh specifies the number of bits to shit the x2APIC ID to get a
```

Comment:
nit: s/0xBh/0xb/, s/bits to shit/bits to shift/

Reply:
fixed